internal/peer/common/common.go

```diff
@@ -144,6 +150,54 @@ func InitCrypto(mspMgrConfigDir, localMSPID, localMSPType string) error {
 	return nil
 }
 
+// Overrides BCCSP config values when corresponding environment variables
+// are set.
+func SetBCCSPConfigOverrides(bccspConfig *factory.FactoryOpts) error {
+	if pkcs11Default, exist := os.LookupEnv("CORE_PEER_BCCSP_DEFAULT"); exist {
+		bccspConfig.Default = pkcs11Default
+	}
+
+	// SW Overrides
+	if swHash, exist := os.LookupEnv("CORE_PEER_BCCSP_SW_HASH"); exist {
+		bccspConfig.SW.Hash = swHash
+	}
+
+	if swSecurity, exist := os.LookupEnv("CORE_PEER_BCCSP_SW_SECURITY"); exist {
+		swSec, err := strconv.Atoi(swSecurity)
+		if err != nil {
+			return errors.Errorf("CORE_PEER_BCCSP_SW_SECURITY set to non-integer value: %s", swSecurity)
+		}
+		bccspConfig.SW.Security = swSec
+	}
```

Comment:
We could remove these override lookups since BCCSP hardcodes these anyways.

Reply:
agreed

Reply:
🧹